File: Auto-generated-tasks/Task-6/Solution.pas
Var
	Count:=4, h: integer;
	R:=0.0: real;
	A:Array[0..Count] of real;
Begin
	for h:= 0 to Count do begin
		R := R + A[h];
	end;
	WriteLn("R=", R);
End.

File: Auto-generated-tasks/Task-6/Solution.php
<?
$Count = 4;
$A = array_fill(0, $Count, 0.0);
$h = 0;
$R = 0.0;
for ($h = 0; $h < $Count; $h++) {
	$R = $R + $A[$h];
}
print("R=" + $R);
?>